docs/topics/db/models.txt

```diff
@@ -259,12 +259,13 @@ details can be found in the :ref:`common model field option reference
 Automatic primary key fields
 ----------------------------
 
-By default, Django gives each model the following field::
+By default, Django gives each model an auto-incrementing primary key with the
+type specified per app in :attr:`AppConfig.default_auto_field
+<django.apps.AppConfig.default_auto_field>` or globally in the
+:setting:`DEFAULT_AUTO_FIELD` setting. For example::
 
     id = models.AutoField(primary_key=True)
```

Comment:
Maybe we could change this to now show `BigAutoField`, in the theme of "encouraging people to use BigAutofield's"?

```suggestion
    id = models.BigAutoField(primary_key=True)
```